mix.exs

```diff
@@ -28,7 +28,7 @@ defmodule Scrivener.Mixfile do
 
   defp deps do
     [
-      {:ecto, "~> 1.0"},
+      {:ecto, ">= 1.0.0 and < 3.0.0"},
```

Comment:
You should specify it as "~> 1.0 or ~> 2.0". If a user want's pre-releases they should override the version requirement with `{:ecto, "~> 2.0.0-beta", override: true}`.